src/components/Dashboard/Tiles/Widgets/WidgetsEditModal.tsx

```diff
@@ -385,6 +386,28 @@ const WidgetOptionTypeSwitch: FC<{
           </Flex>
         </Stack>
       );
+
+    case 'app-select':
+      return (
+        <Stack spacing={0}>
+          <Group align="center" spacing="sm">
+            <Text size="0.875rem" weight="500">
+              {t(`descriptor.settings.${key}.label`)}
+            </Text>
+            {info && <InfoCard message={t(`descriptor.settings.${key}.info`)} link={link} />}
+          </Group>
+          <AppSelector
+            value={value}
+            onChange={(v) => handleChange(key, v ?? option.defaultValue)}
```

Comment:
Rename to event

Reply:
The parameter is the new value, not an event. I'll rename it to `value`